packages/transformers/css-experimental/src/CSSTransformer.js

```diff
@@ -83,7 +83,8 @@ export default (new Transformer({
             symbols: new Map([['*', {local: '*', isWeak: true, loc}]]),
           });
         } else if (dep.type === 'url') {
-          asset.addURLDependency(dep.url, {
+          // if dependency imported with url() includes ", escape " double quotes because this url value will be enclosed with double quote "
+          asset.addURLDependency(dep.url.replace(/"/g, '\\"'), {
```

Comment:
hmm I think this might cause an issue for resolvers because they won't expect pre-escaped values. Perhaps this replacement should be done in the packager. The util [here](https://github.com/parcel-bundler/parcel/blob/9775b9fab1b29d26c655de2aa84d3edc486ca578/packages/core/utils/src/replaceBundleReferences.js#L31) is what does the replacement of placeholders to resolved urls. It might need an option to escape the url.

Reply:
Oh, indeed. As you say, escaping the background image’s URL in Transformer will cause the not found error in resolver. I will fix this :+1: Thank you for your advice!

I think escaping the quote in URL is needed in anyway because
- if URL is not enclosed with quotes, it is not a valid property that the quote exists in the URL
- if URL is enclosed with quotes, the quote should be escaped

https://github.com/parcel-bundler/parcel/pull/7564 as you can see in this PR, I think we need to enclose the URL in quotes and then escape the quotes.

Reply:
I thought about making the resolver able to handle escaped quotes, but I was concerned about performance degradation due to increased computation, so I made a change to remove escaped quotes from URLs with the CSSTransformer and re-escape them with the CSSPackager. [ea56294](https://github.com/parcel-bundler/parcel/pull/7718/commits/ea56294c0c1f7ab09e416276061e3dd993708c29)